transform/cat021_v10_test.go

```diff
@@ -0,0 +1,132 @@
+package transform
+
+import (
+	"math"
+	"testing"
+)
+
+func equalWithinErrorBounds(actualValue float64, targetValue float64, epsilon float64) bool {
+	return math.Abs(targetValue-actualValue) < epsilon
+}
+
+func checkEqualLatLong(resultCoordinates WGS84Coordinates, actualCoordinates WGS84Coordinates, epsilon float64) bool {
```

Comment:
did you see those functions somewhere else in the repo or are these your own? some engineers prefer unit tests to be completely isolated from each other. they don't prefer share functions like this in unit tests. from what I look through the code base, I can't find any private shared functions too. (personally I don't mind) but when contributing to a public repo, probably not a good idea.

Reply:
moved it to common.go